src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -8,10 +8,8 @@
     </None>
   </ItemGroup>
 
-  <!-- MSBuild for .NET Core doesn't support XslTransform at this time -->
   <Target Name="CompileAsn" BeforeTargets="CoreCompile"
-    Condition="'$(MSBuildRuntimeType)' != 'core'"
```

Comment:
Somewhere along the way I ended up with the target file firing when it made no sense to do so.  It might be worth keeping a condition, but instead have

```
Condition=" '@(AsnXml)' != '' "
```

Reply:
Seeing as we both commented at the same time and you may have written this before I posted https://github.com/dotnet/runtime/pull/45220#discussion_r530572836, is my change there sufficient or is is the condition still desirable?